bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/util/Rows.scala

```diff
@@ -31,7 +40,7 @@ trait Rows { this: GridBagPanel =>
     row = row + 1
   }
   def addSpacer() {
-    addRow(new Label(""), new Label("")) // :-/
+    addRow(new Label(" "), new Label("")) // :-/
```

Comment:
This was a bug, or at least did nothing on OS X. Completely empty labels are given a preferred size of (0,0), I believe.